code/user_settings.py

```diff
@@ -0,0 +1,148 @@
+from talon import Module, fs, Context
+import os
+import csv
+from pathlib import Path
+from typing import Dict, List, Tuple
+import threading
+
+
+# NOTE: This method requires this module to be one folder below the top-level
+#   knausj folder.
+SETTINGS_DIR = Path(__file__).parents[1] / "settings"
+
+if not SETTINGS_DIR.is_dir():
+    os.mkdir(SETTINGS_DIR)
+
+
+mod = Module()
+ctx = Context()
+
+
+def _load_csv_dict(
+    file_name: str, headers=Tuple[str, str], default: Dict[str, str] = {}
+) -> Dict[str, str]:
+    """Load a word mapping from a CSV file. If it doesn't exist, create it."""
+    assert file_name.endswith(".csv")
+    path = SETTINGS_DIR / file_name
+
+    # Create the file if it doesn't exist
+    if not SETTINGS_DIR.is_dir():
+        os.mkdir(SETTINGS_DIR)
```

Comment:
since you do this check here, you probably don't need it globally at line 13?

Reply:
I made this trigger-happy to protect against the user deleting the settings dir, but looking at it now you're right, it doesn't make sense. I'll remove this.